src/corpus/corpus.cpp

```diff
@@ -3,9 +3,9 @@
  * @author Sean Massung
  */
 
+#include "meta/corpus/corpus.h"
 #include "cpptoml.h"
 #include "meta/corpus/all.h"
-#include "meta/corpus/corpus.h"
```

Comment:
That's weird. Why would it de-alphabetize the includes?

I'm going to override `clang-format`'s opinion here...